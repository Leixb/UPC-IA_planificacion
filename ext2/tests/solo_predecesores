7

1:3;6:4;7:5
1:3;3:4;5:5;7:6
1:10;3:9;5:7;7:10
