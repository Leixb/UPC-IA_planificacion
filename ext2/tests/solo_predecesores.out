debug files:
 * problem -> /tmp/tmp.Pf5n4Xr900
 * output  -> /tmp/tmp.fwaqpdNiIV
/home/leix/Documents/UPC/IA/lab/UPC-IA_planificacion
/home/leix/Documents/UPC/IA/lab/UPC-IA_planificacion/ext2
compiling parser
running ff-metric ...

ff: parsing domain file
domain 'PLANNER' defined
 ... done.
ff: parsing problem file
problem 'PLANNING1' defined
 ... done.


no metric specified. plan length assumed.

checking for cyclic := effects --- OK.

ff: search configuration is EHC, if that fails then  best-first on 1*g(s) + 5*h(s) where
    metric is  plan length

Cueing down from goal distance:   49 into depth [1]
                                  47            [1]
                                  46            [1]
                                  45            [1]
                                  43            [1]
                                  42            [1]
                                  41            [1]
                                  40            [1]
                                  38            [1]
                                  37            [1]
                                  36            [1]
                                  35            [1]
                                  33            [1]
                                  32            [1]
                                  31            [1]
                                  29            [1]
                                  28            [1]
                                  27            [1]
                                  26            [1]
                                  25            [1]
                                  24            [1]
                                  23            [1]
                                  22            [1]
                                  21            [1]
                                  20            [1]
                                  19            [1]
                                  18            [1]
                                  17            [1]
                                  16            [1]
                                  15            [1]
                                  14            [1]
                                  13            [1]
                                  12            [1]
                                  11            [1]
                                  10            [1]
                                   9            [1]
                                   8            [1]
                                   7            [1]
                                   6            [1]
                                   5            [1]
                                   4            [1]
                                   3            [1]
                                   2            [1]
                                   1            [1]
                                   0            

ff: found legal plan as follows

step    0: PREP-EJERCICIO E3 D1
        1: REALIZAR-EJERCICIO E3 N4 N5 D0 D1 DUMMY
        2: PREP-EJERCICIO E1 D1
        3: PREP-EJERCICIO E3 D2
        4: REALIZAR-EJERCICIO E3 N5 N6 D1 D2 DUMMY
        5: PREP-EJERCICIO E1 D2
        6: REALIZAR-EJERCICIO E1 N3 N4 D0 D1 E3
        7: PREP-EJERCICIO E3 D3
        8: REALIZAR-EJERCICIO E3 N6 N7 D2 D3 DUMMY
        9: PREP-EJERCICIO E1 D3
       10: REALIZAR-EJERCICIO E1 N4 N5 D1 D2 E3
       11: PREP-EJERCICIO E5 D1
       12: REALIZAR-EJERCICIO E5 N5 N6 D0 D1 E1
       13: PREP-EJERCICIO E7 D1
       14: PREP-EJERCICIO E3 D4
       15: REALIZAR-EJERCICIO E3 N7 N8 D3 D4 DUMMY
       16: PREP-EJERCICIO E1 D4
       17: REALIZAR-EJERCICIO E1 N5 N6 D2 D3 E3
       18: PREP-EJERCICIO E5 D3
       19: PREP-EJERCICIO E5 D4
       20: REALIZAR-EJERCICIO E5 N6 N7 D1 D3 E1
       21: PREP-EJERCICIO E7 D3
       22: REALIZAR-EJERCICIO E7 N6 N7 D0 D1 E5
       23: PREP-EJERCICIO E5 D5
       24: PREP-EJERCICIO E7 D4
       25: REALIZAR-EJERCICIO E7 N7 N8 D1 D3 E5
       26: PREP-EJERCICIO E3 D6
       27: PREP-EJERCICIO E3 D7
       28: REALIZAR-EJERCICIO E3 N8 N9 D4 D6 DUMMY
       29: PREP-EJERCICIO E1 D6
       30: REALIZAR-EJERCICIO E1 N6 N7 D3 D4 E3
       31: REALIZAR-EJERCICIO E5 N7 N7 D3 D4 E1
       32: PREP-EJERCICIO E3 D8
       33: REALIZAR-EJERCICIO E3 N9 N9 D6 D7 DUMMY
       34: PREP-EJERCICIO E1 D7
       35: REALIZAR-EJERCICIO E1 N7 N8 D4 D6 E3
       36: REALIZAR-EJERCICIO E5 N7 N7 D3 D5 DUMMY
       37: PREP-EJERCICIO E7 D5
       38: REALIZAR-EJERCICIO E7 N8 N9 D3 D4 E5
       39: REALIZAR-EJERCICIO E3 N9 N9 D6 D8 DUMMY
       40: PREP-EJERCICIO E1 D8
       41: REALIZAR-EJERCICIO E1 N8 N9 D6 D7 E3
       42: REALIZAR-EJERCICIO E7 N9 N10 D4 D5 E5
       43: REALIZAR-EJERCICIO E1 N9 N10 D7 D8 E3
     

time spent:    0.00 seconds instantiating 84128 easy, 0 hard action templates
               0.02 seconds reachability analysis, yielding 1262 facts and 39566 actions
               0.02 seconds creating final representation with 1231 relevant facts, 0 relevant fluents
               0.08 seconds computing LNF
               0.05 seconds building connectivity graph
               0.18 seconds searching, evaluating 65 states, to a max depth of 1
               0.35 seconds total time

# DIA 1 (4 ejercicios)
    - E3	N5
    - E1	N4
    - E5	N6
    - E7	N7
# DIA 2 (2 ejercicios)
    - E3	N6
    - E1	N5
# DIA 3 (4 ejercicios)
    - E3	N7
    - E1	N6
    - E5	N7
    - E7	N8
# DIA 4 (4 ejercicios)
    - E3	N8
    - E1	N7
    - E5	N7
    - E7	N9
# DIA 5 (2 ejercicios)
    - E5	N7
    - E7	N10
# DIA 6 (2 ejercicios)
    - E3	N9
    - E1	N8
# DIA 7 (2 ejercicios)
    - E3	N9
    - E1	N9
# DIA 8 (2 ejercicios)
    - E3	N9
    - E1	N10
# DIA 9 (0 ejercicios)
# DIA 10 (0 ejercicios)
# DIA 11 (0 ejercicios)
# DIA 12 (0 ejercicios)
# DIA 13 (0 ejercicios)
# DIA 14 (0 ejercicios)
# DIA 15 (0 ejercicios)
ASSOLITS
ej 1	N10
ej 3	N9
ej 5	N7
ej 7	N10
